StripeFinancialConnections/StripeFinancialConnections/Source/Analytics/FinancialConnectionsAnalyticsClient.swift

```diff
@@ -22,7 +22,7 @@ final class FinancialConnectionsAnalyticsClient {
     ) {
         self.analyticsClient = analyticsClient
         additionalParameters["is_webview"] = false
-        additionalParameters["navigator_language"] = Locale.current.identifier
+        additionalParameters["navigator_language"] = Locale.current.toLanguageTag()
```

Comment:
not sure if this is even used anywhere, but would only be for analytics anyway:

https://livegrep.corp.stripe.com/search/stripe?q=navigator_language&fold_case=auto&regex=false&context=true